tools/deployment/linux-pkg-postinst.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# we could be called by dpkg or rpm, handle the cases we want to hook
+# or silently claim success
+case "$1" in
+  # debian/ubuntu
+  configure)
```

Comment:
Could this be:
```
configure|2)
```
Then you only `daemon-reload` when `configure` AND `! -z $2`? That should handle the debian and rhel cases?

Matrix as I understand it:
```
debian:
$1=configure       : set on everything
$2=*               : set on upgrade, unset on everything else

rhel:
$1=1               : on install
$1=2               : on upgrade
```